cookiecutter/prompt.py

```diff
@@ -391,7 +391,8 @@ def choose_nested_template(
 
     template = Path(template) if template else None
     if not (template and not template.is_absolute()):
-        raise ValueError("Illegal template path")
+        msg = "Illegal template path"
+        raise ValueError(msg)
```

Comment:
I don't prefer ruff; changes like this, and others here, reinforce that view.

Nevertheless, enforcing style can be valuable, and I'm generally okay with the additional linting.

Reply:
Actually I have no idea why ruff thinks this is a good idea, but OTOH it does not hurt too.

Reply:
this lint is because python prints both the error message and the context. When the context also contains the message this is redundant.

ie. this would print something like (paraphrasing)

```
error: Illegal template path
`raise ValueError("Illegal template path")`
```

but if the msg is defined outside the context you get

```
error: Illegal template path
`raise ValueError(msg)`
```

in the trace